### PR TITLE
Enable cache-deployer as fixed the root cause in other PR

### PR DESCRIPTION
**Pull request by rmgogogo**
Enable cache-deployer as fixed the root cause in other PR

### CODE REVIEW

**Comment by Bobgy** [comment]
I found the root cause is for cache-server
```
        message: configmap "cache-configmap" not found
        reason: CreateContainerConfigError
```

**Comment by Bobgy** [comment]
@rmgogogo why was the configmap removed in https://github.com/kubeflow/pipelines/commit/91c323900435cd3013549894e4afd8ed14523407#diff-22261554aaee8ffbc4ba68a2ca4c4fd8?

**Comment by rmgogogo** [comment]
> Turns out deployer worked properly, but @rmgogogo why was the configmap removed in [91c3239#diff-22261554aaee8ffbc4ba68a2ca4c4fd8](https://github.com/kubeflow/pipelines/commit/91c323900435cd3013549894e4afd8ed14523407#diff-22261554aaee8ffbc4ba68a2ca4c4fd8)?

I extracted a common mysql-config and mysql-secret for the usage.

For the error log you see, not sure which version you see it. It's possible due to namespace (I default env/gcp to kubeflow

**Comment by Bobgy** [comment]
@rmgogogo I found the root cause, when you were doing the refactoring, you forgot to remove one usage of cache-configmap: https://github.com/kubeflow/pipelines/commit/91c323900435cd3013549894e4afd8ed14523407

**Comment by rmgogogo** [comment]
> @rmgogogo I found the root cause, when you were doing the refactoring, you forgot to remove one usage of cache-configmap: [91c3239](https://github.com/kubeflow/pipelines/commit/91c323900435cd3013549894e4afd8ed14523407)

thanks. yes, I introduced another issue when fix previous issue, ;-)

**Comment by Bobgy** [comment]
@rmgogogo I verified cache service is working properly with the fix.

File: backend/src/cache/deployer/deploy-cache-service.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ set -ex
 
 echo "Start deploying cache service to existing cluster:"
 
-NAMESPACE=${NAMESPACE_TO_WATCH:-default}
+NAMESPACE=${NAMESPACE_TO_WATCH:-kubeflow}
```
Comment (rmgogogo) [inline backend/src/cache/deployer/deploy-cache-service.sh:25]: it's not necessary as external deployment passed in the value.
it's more for better default value

**Comment by rmgogogo** [comment]
> @rmgogogo I verified cache service is working properly with the fix.

Thanks. Too many test now and hit "message: u'The network "default" does not have available private IP space in 10.0.0.0/8 to reserve a /14 block for containers for cluster". Will retest a little bit later.

May leave Alexy to approve it as Rui still has one PR for cache-lifetime. assume that one won't block this PR but better let Alexy decide whether release without that.

File: backend/src/cache/deployer/deploy-cache-service.sh (hunk 1 above)
Comment (Ark-kun) [reply backend/src/cache/deployer/deploy-cache-service.sh:25]: Does any deployment method use `default` namespace?

**Comment by Ark-kun** [comment]
> May leave Alexy to approve it as Rui still has one PR for cache-lifetime. assume that one won't block this PR but better let Alexy decide whether release without that.

It's fine to release without that PR (it's a P2 feature that I'll finish one day).
/lgtm
/approve

**Pull request merged**